daemon/api_test.go

```diff
@@ -152,11 +158,19 @@ func (s *apiBaseSuite) SetUpSuite(c *check.C) {
 	s.restoreRelease = release.MockForcedDevmode(false)
 
 	snapstate.CanAutoRefresh = nil
+	s.origSysctlcmd = systemd.SystemctlCmd
+	systemd.SystemctlCmd = s.systemctl
```

Comment:
`systemd` exports `SystemctlCmd` so you can overwrite it in tests like this without having to mock the executable itself